packages/web-components/src/components/va-text-input/va-text-input.scss

```diff
@@ -64,48 +64,4 @@ input.usa-input {
   #inputField {
     padding-left: 25px;
   }
-}
-
-
-/** Original Component Style **/
-@import '../../mixins/accessibility.css';
-@import '../../mixins/form-field-error.css';
```

Comment:
Is there anything we can remove from the `form-field-error.css` mixin yet? It looks like there's some v1 specific styles there. 

https://github.com/department-of-veterans-affairs/component-library/blob/655db88c85816a4c153305b9229548d462dc835d/packages/web-components/src/mixins/form-field-error.css#L25-L35

Reply:
I think textarea could probably be removed from that, but I would be more comfortable removing it after the rest of the v1 code is removed.